digitalocean/app/app_spec.go

```diff
@@ -189,6 +188,40 @@ func appSpecAppLevelAlerts() *schema.Resource {
 				Default:  false,
 				Optional: true,
 			},
+			"notifications": {
```

Comment:
We call this destinations in the HTTP API. It probably makes sense to use the same terminology here.

```suggestion
			"destinations": {
```